File: src/cdva/tests/fixtures/go/multibyte.go
package greet

import "testing"

// Greet greets in Japanese.
func Greet() string {
	return "こんにちは"
}

func TestGreet(t *testing.T) {
	if Greet() != "こんにちは" {
		t.Fatalf("挨拶が違う: %s", Greet())
	}
}

File: src/cdva/tests/fixtures/ruby/multibyte.rb
# Greeter greets in Japanese.
class Greeter
  def greet
    "こんにちは"
  end
end

RSpec.describe Greeter do
  it "日本語で挨拶する" do
    expect(Greeter.new.greet).to eq("こんにちは")
  end
end
